frontend/src/pages/DashboardPage/ChartComponenet/ChartComponent.jsx

```diff
@@ -145,7 +145,7 @@ const ChartComponent = forwardRef(({ chartData, sliderValue }, ref) => {
               label: (tooltipItem) => {
                 const item = accuracyData[tooltipItem.dataIndex];
                 return [
-                  `Accuracy: ${item.accuracy}`,
+                  `Bias: ${1 - item.accuracy}`,
```

Comment:
Did you check with Akshat about the label of y_axis

Reply:
Yes I checked with him